### PR TITLE
chore: use official download url for pi install

### PR DESCRIPTION
**Pull request by bumi**
(still need to test it)

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@ echo "Installing..."
 sudo mkdir -p /opt/albyhub
 sudo chown -R $USER:$USER /opt/albyhub
 cd /opt/albyhub
-wget https://github.com/getAlby/nostr-wallet-connect-next/releases/latest/download/albyhub-Server-Linux-armv6.tar.bz2
+wget https://getalby.com/install/hub/server-linux-armv6
```
Comment (reneaaron) [inline scripts/install.sh:11]: This download gets you a file named `server-linux-armv6` (without any file endings). Do you plan to add some file ending?

In any case we need to adjust the paths in the script...

Comment (reneaaron) [reply scripts/install.sh:11]: I updated the paths, let me know if you change the downloaded file's name.

**Review by reneaaron — APPROVED** [review]
tACK

File: scripts/install.sh (hunk 1 above)
Comment (bumi) [reply scripts/install.sh:11]: I am just deploying a version where we can specify the file ending in the download url.
to use `https://getalby.com/install/hub/server-linux-armv6.tar.bz2`

what is better?

Comment (reneaaron) [reply scripts/install.sh:11]: That's nice, I think the files should have proper file extensions. I'll update the script :+1:

File: scripts/install.sh [hunk 2]
```diff
@@ -8,13 +8,13 @@ echo "Installing..."
...
```
Comment (bumi) [inline scripts/install.sh:11]:
```suggestion
wget https://getalby.com/install/hub/server-linux-armv6.tar.bz2
```

File: scripts/update.sh [hunk 3]
```diff
@@ -9,14 +9,14 @@ rm -rf albyhub-backup
 mkdir albyhub-backup
 mv bin albyhub-backup
 cp -r data albyhub-backup
-wget https://github.com/getAlby/nostr-wallet-connect-next/releases/latest/download/albyhub-Server-Linux-armv6.tar.bz2
+wget https://getalby.com/install/hub/server-linux-armv6
```
Comment (bumi) [inline scripts/update.sh:12]:
```suggestion
wget https://getalby.com/install/hub/server-linux-armv6.tar.bz2
```

**Comment by reneaaron** [comment]
@bumi Just saw you updated the download links as well. 

I thought of these more like static links (so we don't require content editors to memorize the file extensions).

**Comment by reneaaron** [comment]
ℹ️ File extensions are not required, but are added as wget saves the file by the path you provide

**Pull request merged**